app/routes/dashboards/dashboard.js

```diff
@@ -571,7 +571,7 @@ export default Ember.Route.extend({
                             },
                             {
                                 parameterPath: ["query", "bool", "should"],
-                                defaultValue: (()=>{ return transition.queryParams.all ? ucsd_query : undefined; })()
+                                defaultValue: ucsd_query
```

Comment:
filter by ucsd query by default. curated is now a filter on that; ucsd source (curated) is a subset of ucsd all